DVTShowcaseColor.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  DVTShowcaseColor.swift
+//  DVTShowcase
+//
+//  Created by Idris SOP on 2017/09/18.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+import UIKit
+
+extension UIColor {
```

Comment:
It's not wise to put an extension for a class into a file with another name. This makes it difficult to find the extension if one is looking for it. Rather place the extensions to UIColor into a file called "UIColor+Extensions.swift" or something similar.